### PR TITLE
Update/modules

### PR DESCRIPTION
**Pull request by grantshepert**

### CODE REVIEW

File: 04_front-end-layout-manager/01_introduction/01_layout-manager.md [hunk 1]
```diff
@@ -10,6 +10,22 @@ permalink: /front-end-layout-manager/introduction/layout-manager/
 
 # Layout Manager
 
+The Layout Manager is the front-end tool for composing the layout of pages, entering content, and adding/configuring automated tools like navigation, forms, feeds and collections.
+
 ## Concept
 
+Instead of relying upon hard-coded, code-based layout templates, the Layout Manager allows you to drag-and-drop highly configurable "Modules"  onto a page. Each Module (depending upon it's type and location) will have an associated "configurator" that allows further customization, such as functional settings (feed source, form name, etc.) and layout (alignment/width, associated CSS classes/styles, etc.). In this way, the editor is allowed a great deal of flexibility when laying out each page, seeing the results immeditately and without having to edit or create hard-coded templates.
```
Comment (guustnieuwenhuis) [inline 04_front-end-layout-manager/01_introduction/01_layout-manager.md:17]: double space

File: 04_front-end-layout-manager/01_introduction/01_layout-manager.md [hunk 2]
```diff
@@ -10,6 +10,22 @@ permalink: /front-end-layout-manager/introduction/layout-manager/
...
+
 ## How-to
+
+To enable the Layout Manager, log into Masa with an account that has editing permissions enabled, then from the top editor toolbar choose "Edit (pencil icon) -> Inline Edit". A list of Modules will appear on the right-hand side of the page. Use your mouse to drag this into the content area. You will see a context label appear above an available "drop" container when you drag the Module over it. Releasing the mouse button (aka "dropping" it) will place the Module into the container.
```
Comment (guustnieuwenhuis) [inline 04_front-end-layout-manager/01_introduction/01_layout-manager.md:21]: Masa CMS

File: 04_front-end-layout-manager/02_modules/01_content.md [hunk 3]
```diff
@@ -10,6 +10,24 @@ permalink: /front-end-layout-manager/modules/content/
...
+To edit content inline, simply double-click on the text. For block content, a simplified editor menu will appear above the content. You can simply type in place to add or update the content. Clicking outside of the Content area (or clicking on "Done Editing" in the Layout Manager) will close the editing process, and once the page is saved the change is implemented.
+
+## Via Text Module
+A Content module can also be dragged onto the page. These allow you to create content blocks of three types: Custom, Component, and Bound Attribute. You can select this via the Text Module's "Content Source" setting it it's Configurator.
+
+### Custom
+Custom content is similar to the page's "body" content, in that you use the Masa HTML editor to create a new content block. By selecting "Custom" as the Content Source, and then clicking on the "Edit" button, an Editor modal will appear in which you can create or edit the content.
+
+### Component
+Component allows you to select or create a new Masa Component. Components are shared, reusable content blocks, so remember that editing an existing component will change the content wherever it is displayed.
```
Comment (guustnieuwenhuis) [inline 04_front-end-layout-manager/02_modules/01_content.md:27]: Masa CMS

File: 04_front-end-layout-manager/02_modules/02_collections.md [hunk 4]
```diff
@@ -8,6 +8,20 @@ last_modified_date: 20-10-2021 13:00
...
-# Collections
+# Collection
 
-## Types
+A Collection Module places a Masa "Collection" onto the page. Collections are generated in a number of ways within Masa, each type being available under the "Content Source" Configurator dropdown. Depending upon which Content Source is selected, a selection of related options will appear within the Configurator.
+
+## Types (Content Source)
+
+### Local Index
+A Local Index is a Collection of content you have created within the Masa Administrator.
```
Comment (guustnieuwenhuis) [inline 04_front-end-layout-manager/02_modules/02_collections.md:18]: Masa CMS

File: 04_front-end-layout-manager/02_modules/07_plugins.md [hunk 5]
```diff
@@ -8,4 +8,7 @@ last_modified_date: 20-10-2021 13:00
 permalink: /front-end-layout-manager/modules/plugins/
 ---
 
-# Plugins
\ No newline at end of file
+# Plugins
+A Masa Plugin may have it's own set of Modules (sometimes refered to as "display objects") to render it's front-end content. If a plugin has Modules/Display Objects associated with it, then these will be available within the Plugin's Configurator.
```
Comment (guustnieuwenhuis) [inline 04_front-end-layout-manager/02_modules/07_plugins.md:12]: Masa CMS

File: 04_front-end-layout-manager/02_modules/16_system.md [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+title: System
+nav_order: 7
+grand_parent: Front-End/Layout Manager
+parent: Modules
+last_modified_date: 20-10-2021 13:00
+permalink: /front-end-layout-manager/modules/system/
+---
+
+# System
+System places one of the Masa "system" Modules onto the page.
```
Comment (guustnieuwenhuis) [inline 04_front-end-layout-manager/02_modules/16_system.md:12]: Masa CMS

**Review by guustnieuwenhuis — CHANGES_REQUESTED** [review]
Add a label to the top of the page for the new features in 7.4. As done on this page for 7.3: https://docs.masacms.com/front-end-layout-manager/m-tag/masa-scope/

**Comment by grantshepert** [comment]
Updated, new content, changes implemented.

**Review by guustnieuwenhuis — APPROVED** [review]

**Pull request merged**